### PR TITLE
Targeted bugfixes for develop branch

### PR DESCRIPTION
**Pull request by Lego6245**
1) Fixes problem where either the addedCards or aetherized cards were null.

2) Fixes sort not working in explore page (we need to refactor it so that it can sort without executing a new search every time)

### CODE REVIEW

**Comment by Thaoden** [comment]
The addedCards / aetherizedCards is adressed in #703 already.

**Comment by Lego6245** [comment]
#703’s change feels a little messier to me. I’m willing to drop the fix from this changeset though

File: src/window_main/FilterPanel.tsx [hunk 1]
```diff
@@ -127,7 +127,7 @@ class FilterPanel {
               <ManaFilter prefixId={this.prefixId} filterKey={"colors"} filters={this.filters} onFilterChanged={this.filterCallback("colors")} />
             )}
         </div>)}
-        {(this.archs.length || this.showOppManaFilter) && (
+        {(this.archs.length || this.showOppManaFilter || this.showSortOption) && (
           <div className={"columnC"}>
             {this.archs.length && <WrappedReactSelect className={this.prefixId + "_query_optag filter_panel_select_margin spacer180"} options={this.archs} current={this.filters.arch} callback={this.filterCallback('arch')} optionFormatter={(tag: string) => <TagOption tag={tag} showCount={true} archCounts={this.archCounts} />} />}
```
Comment (Manuel-777) [inline src/window_main/FilterPanel.tsx]: I believe this should be an if? 
`this.archs.length ? <`

The way it is now adds a "0" before the sort selector in the DOM.

**Comment by Lego6245** [comment]
Good catch!

**Pull request merged**